pkg/workspaces/workspaces_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2019 The Tekton Authors.
+// Copyright © 2020 The Tekton Authors.
```

Comment:
I think the way to update this is actually `2019-2020` per this [comment](https://github.com/tektoncd/cli/issues/621#issuecomment-577676677).

Reply:
Thanks for pointing to the comment.